firefox-ios/firefox-ios-tests/Tests/ClientTests/Utils/StoreTestUtility.swift

```diff
@@ -8,26 +8,26 @@ import Redux
 
 protocol StoreTestUtility {
     func setupAppState() -> AppState
-    func setupTestingStore()
-    func resetTestingStore()
+    func setupStore()
+    func resetStore()
 }
 
 /// Utility class used when replacing the global store for testing purposes
 class StoreTestUtilityHelper {
-    static func setupTestingStore(with appState: AppState, middlewares: [Middleware<AppState>]) {
+    static func setupStore(with appState: AppState, middlewares: [Middleware<AppState>]) {
```

Comment:
I will create a separate ticket to address the other middleware tests and remove this if not needed.